test/development/middleware-errors/index.test.ts

```diff
@@ -124,35 +156,39 @@ describe('middleware - development errors', () => {
       await retry(() => {
         expect(stripAnsi(next.cliOutput)).toContain('Dynamic Code Evaluation')
       })
-      // TODO: assert on full, ignore-listed stack
       if (isTurbopack) {
         // Locally, prefixes the "test is not defined".
         // In CI, it prefixes "Dynamic Code Evaluation".
         expect(stripAnsi(next.cliOutput)).toContain(
-          '\n ⚠ middleware.js (3:22) @ __TURBOPACK__default__export__' +
-            '\n ⨯ middleware.js (4:9) @ eval'
+          // TODO(veil): Should be sourcemapped
+          '\n    at __TURBOPACK__default__export__ (.next/'
```

Comment:
I would love to have these turbopack special identifiers demangled before printing. There is a method to demangle them in a string we need to call
```suggestion
          '\n    at {default export} (.next/'
```

Reply:
My current thinking is that these should just be sourcemapped to `default`. That's what you'd get in the original stack trace.

But we already special case the WEBPACK__default__export so I probably just go with that   short term.

Reply:
packages/next/src/shared/lib/magic-identifier.ts decodeMagicIdentifier